### ISSUE TITLE
It does not reach the mAP38.5 in the paper.

### ISSUE THREAD

**Issue by JYJAI** [sixiaoqi77/uav#2]
I ran the experiment exactly according to the parameters in the paper, but the result did not reach the score written in the paper.

**Comment by sixiaoqi77** [comment]
Can I ask if you downloaded the version of the network provided by my github and the method described in the article to train it? Is the image size set to the training size when the model is tested?

**Comment by JYJAI** [comment]
Can you give me your weight file

**Comment by JYJAI** [comment]
train：
--data
coco128.yaml
--weights
yolov5s.pt
--img
960

**Comment by JYJAI** [comment]
lr0: 0.01 # initial learning rate (SGD=1E-2, Adam=1E-3)
lrf: 0.01 # final OneCycleLR learning rate (lr0 * lrf)
momentum: 0.937 # SGD momentum/Adam beta1
weight_decay: 0.0005 # optimizer weight decay 5e-4
warmup_epochs: 3.0 # warmup epochs (fractions ok)
warmup_momentum: 0.8 # warmup initial momentum
warmup_bias_lr: 0.1 # warmup initial bias lr
box: 0.05 # box loss gain
cls: 0.5 # cls loss gain
cls_pw: 1.0 # cls BCELoss positive_weight
obj: 1.0 # obj loss gain (scale with pixels)
obj_pw: 1.0 # obj BCELoss positive_weight
#iou_t: 0.20 # IoU training threshold
iou_t: 0.50 # IoU training threshold
anchor_t: 4.0 # anchor-multiple threshold
# anchors: 3  # anchors per output layer (0 to ignore)
fl_gamma: 0.0 # focal loss gamma (efficientDet default gamma=1.5)
hsv_h: 0.015 # image HSV-Hue augmentation (fraction)
hsv_s: 0.7 # image HSV-Saturation augmentation (fraction)
hsv_v: 0.4 # image HSV-Value augmentation (fraction)
degrees: 0.0 # image rotation (+/- deg)
translate: 0.1 # image translation (+/- fraction)
scale: 0.5 # image scale (+/- gain)
shear: 0.0 # image shear (+/- deg)
perspective: 0.0 # image perspective (+/- fraction), range 0-0.001
flipud: 0.0 # image flip up-down (probability)
fliplr: 0.5 # image flip left-right (probability)
mosaic: 1.0 # image mosaic (probability)
mixup: 0.0 # image mixup (probability)
copy_paste: 0.0 # segment copy-paste (probability)

**Comment by sixiaoqi77** [comment]
These are just some of the relevant parameters for model training, and the final model obtained from the training is related to many factors, such as the different parameters of the convolution kernel set in the configuration file can also lead to differences in the final results.

**Comment by sixiaoqi77** [comment]
I can give you the relevant weights file, what is your email address please?

**Comment by JYJAI** [comment]
jyj1245179883@163.com

**Comment by JYJAI** [comment]
Can you provide me with more details about the experiment

**Comment by JYJAI** [comment]
Model Summary: 213 layers, 7037095 parameters, 0 gradients, 15.8 GFLOPs
               Class     Images     Labels          P          R     mAP@.5 mAP@.5:.95: 100%|██████████| 18/18 [00:10<00:00,  1.67it/s]
                 all        548      38759      0.554      0.442       0.46      0.271
          pedestrian        548       8844      0.624      0.478      0.528      0.238
              people        548       5125      0.543      0.361      0.397      0.149
             bicycle        548       1287      0.374       0.25      0.248       0.11
                 car        548      14064      0.751      0.816      0.838      0.593
                 van        548       1975      0.542      0.478      0.486       0.34
               truck        548        750      0.557      0.408      0.424      0.285
            tricycle        548       1045      0.462       0.37      0.341      0.191
     awning-tricycle        548        532      0.336      0.203      0.196      0.125
                 bus        548        251      0.757      0.546      0.629       0.45
               motor        548       4886      0.594      0.505      0.514      0.227

**Comment by sixiaoqi77** [comment]
Suggest you check if the network structure is the same as ours, and also go over the specifics of how the dataset is used.

**Comment by JYJAI** [comment]
> 我可以给你相关的权重文件，请问你的电子邮件地址是什么？

When will you give me the weights file?

**Comment by JYJAI** [comment]
> I can give you the relevant weights file, what is your email address please?

When will you give me the weights file?

**Comment by JYJAI** [comment]
I have reproduced the experiment according to the parameters in the paper, but the problem is not solved. Why close it?
